packages/palette-charts/src/elements/BarChart/Bar.tsx

```diff
@@ -12,14 +20,19 @@ const BAR_HEIGHT_RANGE = MAX_BAR_HEIGHT - MIN_BAR_HEIGHT
 
 interface BarBoxProps {
   isHighlighted?: boolean
+  primaryColor?: Color
+  hoverColor?: Color
+  highlightColor?: Color
 }
 
 // the actual visible bit of the bar
 const BarBox = styled(Box)<BarBoxProps>`
   transition: height 0.8s ease;
   position: relative;
-  background: ${(props: BarBoxProps) =>
-    props.isHighlighted ? color("mono60") : color("mono10")};
```

Comment:
I guess charts are rare enough that no one's really noticed this before.

Reply:
I think it's one and the same, if we accept custom colors we need to make sure it works in dark mode

Reply:
To be clear: just by using `themeGet` everything will work automatically